[Docs] Adding move-analyzer VS Code plugin link. (#1772)
aptos-labs/aptos-core: developer-docs-site/docs/guides/getting-started.md

Now your basic Aptos development environment ready.

> ## Move plugins

Reviewer: ```suggestion
## IDE plugins for Move
```

> - A [Move language syntax plugin](https://marketplace.visualstudio.com/items?itemName=damirka.move-syntax) for Visual Studio Code.

Reviewer: ```suggestion
- [Syntax Hightlighting for VS Code](https://marketplace.visualstudio.com/items?itemName=damirka.move-syntax)
```

> - A [Visual Studio Code plugin for Move, called move-analyzer](https://marketplace.visualstudio.com/items?itemName=move.move-analyzer).

Reviewer: ```suggestion
- [Move-Analyzer for VS Code](https://marketplace.visualstudio.com/items?itemName=move.move-analyzer) - supports advanced code navigation and syntax highlighting
```

Author: The term `move-analyzer` is the name of the plugin, and we should use the term as it is. Changing the above suggested `Move-Analyzer` to `Move analyzer` so now this is just a label.
